components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,10 +1,12 @@
 import { Fragment } from "react";
-
-export default function Breadcrumbs({ list }) {
+type props = {
+  list: any;
+};
```

Comment:
`props` is lower-cased, but should be in `PascalCase`. Also, is this maybe this type:
```suggestion
type ListItem = {
  url: string;
  text: string;
}

type Props = {
  list: ListItem[];
};
```

Reply:
done